test/actions/store_test.exs

```diff
@@ -18,6 +18,15 @@ defmodule WaffleTest.Actions.Store do
     def __versions, do: [:original, :thumb, :skipped]
   end
 
+  defmodule DummyDefinitionWithHeaders do
+    use Waffle.Actions.Store
+    use Waffle.Definition.Storage
+
+    def transform(_, _), do: :noaction
+    def __versions, do: [:original, :thumb, :skipped]
+    def remote_file_headers(_), do: [{"User-Agent", "MyApp"}]
```

Comment:
we have a warning on that line

```
warning: this clause cannot match because a previous clause at line 27 always matches
  test/actions/store_test.exs:21
```